integration/internal/swarm/service.go

```diff
@@ -19,10 +19,10 @@ import (
 // ServicePoll tweaks the pollSettings for `service`
 func ServicePoll(config *poll.Settings) {
 	// Override the default pollSettings for `service` resource here ...
-
+	config.Timeout = 30 * time.Second
```

Comment:
Would be nice to use `defaultReconcilliationTimeout`; else value looks arbitrary.

Reply:
I think these values are pretty arbitrary, but so is defaultReconciliationTimeout. They're just values that happen to work for CI.